packages/agreement-process/src/model/domain/errors.ts

```diff
@@ -10,12 +11,14 @@ const errorCodes = {
   agreementNotInExpectedState: "0003",
   descriptorNotInExpectedState: "0004",
   operationNotAllowed: "0005",
+  agreementDocumentAlreadyExists: "0006",
```

Comment:
Is this error from the old `management` service?
If so we need to remap the error code, the old `agreement-process` service defines this kind of errors:
```typescript
const errorCodes = {
  missingCertifiedAttributesError: "0001",
  agreementSubmissionFailed: "0002",
  agreementNotInExpectedState: "0003",
  descriptorNotInExpectedState: "0004",
  eServiceNotFound: "0005",
  submitAgreementError: "0006",
  operationNotAllowed: "0007",
  agreementActtivationFailed: "0008",
  agreementNotFound: "0009",
  agreementAlreadyExists: "0010",
  noNewerDescriptor: "0011",
  publishedDescriptorNotFound: "0012",
  unexpectedVersionFormat: "0013",
  descriptorNotFound: "0014",
  stampNotFound: "0015",
  missingUserInfo: "0016",
  documentNotFound: "0017",
  documentChangeNotAllowed: "0018",
  selfcareIdNotFound: "0019",
  tenantIdNotFound: "0020",
  notLatestEServiceDescriptor: "0021",
  attributeNotFound: "0022",
  invalidAttributeStructure: "0023",
  consumerWithNotValidEmail: "0024",
};
```
The 0006 error code should be: `submitAgreementError`

Reply:
I have applied all error codes defined in your [PR](https://github.com/pagopa/interop-be-monorepo/pull/120)